samples/app/src/main/AndroidManifest.xml

```diff
@@ -94,5 +94,8 @@
             </intent-filter>
         </receiver>
 
+        <meta-data
```

Comment:
Note keys can also be added to fabric.properties so that they do not appear in public repo.

```
apiKey=0000000000000000000
apiSecret=00000000000000000000000000000000000000
twitterConsumerKey=0000000000000000000
twitterConsumerSecret=00000000000000000000000000000000000000
```